### ISSUE TITLE
Initial change/collapse not recognized

### ISSUE THREAD

**Issue by aaronfay** [UAlbertaALTLab/cree-intelligent-dictionary#928]
In the case of this collapsed preverb `ka-` itwewina is doing the right thing:

```
k-âcimostawak (ka-âcimostawak)
```
![screenshot 2021-07-23 at 21 29 41](https://user-images.githubusercontent.com/1033941/126857412-ac80bb75-7343-41c7-a37e-5e02a9457323.png)

However with initial change in the mix, this collapsed preverb combo does not get recognized:

```
k-êsi-pimâcihocik (kê-isi-pimâcihocik)
```
(reference kwayask ê-kî-pê-kiskinowâpahtihicik p18-13)

![screenshot 2021-07-23 at 21 34 15](https://user-images.githubusercontent.com/1033941/126857421-5e809fe1-4834-4d0a-87f3-98e6574029c7.png)

**Comment by aarppe** [comment]
We changed spelling relaxation in the analyzing FST so that a preverb-final vowel can be dropped in all circumstances (If that somehow results from initial change, that shouldn't matter, but I'm yet checking). So, with the latest FST, I do get the following analyses:

```
hfst-lookup -q src/analyser-gt-relaxed.hfst
k-isi-pimâcihocik
k-isi-pimâcihocik	PV/ki+PV/isi+pimâcihow+V+AI+Cnj+3Pl+Err/Fra0.000000
k-isi-pimâcihocik	PV/ka+PV/isi+pimâcihow+V+AI+Cnj+3Pl	0.000000
k-isi-pimâcihocik	PV/ke+PV/isi+pimâcihow+V+AI+Cnj+3Pl	0.000000
```

... and it appears to work in itwêwina, cf. https://itwewina.altlab.app/search?q=k-isi-pim%C3%A2cihocik

<img width="420" alt="image" src="https://user-images.githubusercontent.com/15003407/127074623-2ea777f2-adb8-45d9-b642-f87b581a0a52.png">

**Comment by aaronfay** [comment]
Is it likely then @aarppe that `-êsi-` would need to be recognized as a preverb in order for Itwewina to recognize `k-êsi-pimâcihocik`?

**Comment by aarppe** [comment]
Yes, _êsi-_ would need to be specified as a preverb in the LEXC source, for verb forms with _êsi-_ to be recognized.

**Comment by aaronfay** [comment]
Here's another example involving initial change that isn't recognized by itwewina:

```
nik-êtwân  "I will say"
```
(kwayask ê-kî-pê-kiskinowâpahtihicik p44-33)

Presumably a collapsation of `nika-itwân`, but possibly `nikê-itwân`?

![screenshot 2021-08-05 at 10 59 11](https://user-images.githubusercontent.com/1033941/128398605-fca4e12e-32e1-460e-a9c5-4beeb2b8ebf4.png)
